Lesson6/main.js

```diff
@@ -0,0 +1,25 @@
+const ul = document.getElementById("js-ul");
+const fragment = new DocumentFragment();
+const features = [{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+{to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}];
+
+const checkFeatures = new Promise((resolve) => {
```

Comment:
IMO
In this case, I felt it was not necessary to put it in a variable. Because it has not been used since then. You can refer to error messages in the code sandbox. 
![スクリーンショット 2022-09-26 21 06 31](https://user-images.githubusercontent.com/51282804/192272239-d740049e-1892-41ab-9a00-7623ad87a7d7.png)